docs/guide.pug

```diff
@@ -167,22 +167,22 @@ block content
 
     ```javascript
       // define a schema
-      var animalSchema = new Schema({ name: String, type: String });
+      const animalSchema = new Schema({ name: String, type: String });
 
       // assign a function to the "methods" object of our animalSchema
-      animalSchema.methods.findSimilarTypes = function(cb) {
-        return mongoose.model('Animal').find({ type: this.type }, cb);
-      };
+      animalSchema.methods.findSimilarTypes = cb => (
```

Comment:
This change doesn't work because the method uses `this`, which [arrow functions](https://masteringjs.io/tutorials/fundamentals/arrow) mess up.